mediator/med_phases_aofluxes_mod.F90

```diff
@@ -877,23 +879,26 @@ subroutine med_aofluxes_init_xgrid(gcomp, aoflux_in, aoflux_out, rc)
     dataptr(:) = 1.0_r8
 
     ! create agrid->xgrid route handles
-    call ESMF_FieldRegridStore(xgrid, field_a, field_x, routehandle=rh_agrid2xgrid, rc=rc)
+    call ESMF_FieldRegridStore(xgrid, field_a, field_x, routehandle=rh_agrid2xgrid, &
+         srcTermProcessing=srcTermProcessing_Value, rc=rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
     call ESMF_FieldRegridStore(xgrid, field_a, field_x, routehandle=rh_agrid2xgrid_2ndord, &
-         regridmethod=ESMF_REGRIDMETHOD_CONSERVE_2ND, rc=rc)
+         regridmethod=ESMF_REGRIDMETHOD_CONSERVE_2ND, srcTermProcessing=srcTermProcessing_Value, rc=rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
     if (trim(coupling_mode) == 'cesm') then
-       stp = 1
```

Comment:
@jedwards4b - particularly note that I changed this value from 1 to 0 even for places where you were already setting srcTermProcessing. My sense is that you chose a value of 1 based on guidance from @oehmke (based on looking back at some email exchanges from last year), and in a conversation with @theurich and @oehmke yesterday they supported using 0 instead of 1. But let me know if you had reasons for choosing a value of 1 here that we should maintain. Note that we use a value of 0 in med_map_mod. According to @theurich, both 0 and 1 ensure bit-for-bit reproducibility, but he feels that 0 is probably better for performance in many cases.